src/status_im/common/lightbox/effects.cljs

```diff
@@ -15,10 +14,11 @@
    (blob/fetch uri
                config
                (fn [downloaded-url]
-                 (share/open {:url       (str (when platform/android? "file://") downloaded-url)
-                              :isNewTask true}
-                             #(fs/unlink downloaded-url)
-                             #(fs/unlink downloaded-url))))))
+                 (rf/dispatch [:open-share
+                               {:url       (str (when platform/android? "file://") downloaded-url)
+                                :isNewTask true}
+                               {:on-success #(fs/unlink downloaded-url)
+                                :on-error   #(fs/unlink downloaded-url)}])))))
```

Comment:
Some of these changes are taking something that is being called synchronously `(share/open)` and turning it into a delayed call via dispatch. Perhaps this should be `dispatch-sync` thoughts?

Reply:
Isn't `share/open` asynchronous? I thought it returned a promise

Reply:
You're right `(share/open) is asynchronous, so this might not be a concern 👍